src/module_wrap.cc

```diff
@@ -63,6 +62,66 @@ using v8::UnboundModuleScript;
 using v8::Undefined;
 using v8::Value;
 
+ModulePhase to_phase_constant(ModuleImportPhase phase) {
+  switch (phase) {
+    case ModuleImportPhase::kEvaluation:
+      return kEvaluationPhase;
+    case ModuleImportPhase::kSource:
+      return kSourcePhase;
+  }
+  UNREACHABLE();
+}
+
+void ModuleRequest::MemoryInfo(MemoryTracker* tracker) const {
+  tracker->TrackField("specifier", specifier);
+  tracker->TrackField("import_attributes", import_attributes);
+  tracker->TrackField("phase", static_cast<int>(phase));
+}
+
+template <int elements_per_attribute>
```

Comment:
Why are there `3` elements per attribute? What is the third element?

Reply:
Well, like said, there are two conditions of number of elements (3 for static, 2 for dynamic). So better to yield to the caller to decide how many number of elements of the import attributes.

Reply:
Oh right for dynamic import call handling. Got it thanks.